lib/ansible/module_utils/redfish_utils.py

```diff
@@ -0,0 +1,932 @@
+            return {'ret': True}
+
+    def _find_managers_resource(self, uri):
+        response = self.get_request(self.root_uri + uri)
+        if response['ret'] is False:
+            return response
+        data = response['data']
+        if 'Managers' not in data:
+            return {'ret': False, 'msg': "Manager resource not found"}
+        else:
+            manager = data["Managers"]["@odata.id"]
+            response = self.get_request(self.root_uri + manager)
+            if response['ret'] is False:
+                return response
+            data = response['data']
+            for member in data[u'Members']:
+                manager_service = member[u'@odata.id']
+            self.manager_uri = manager_service
+            return {'ret': True}
+
+    def get_logs(self):
+        log_svcs_uri_list = []
+        list_of_logs = []
+        result = {}
+
+        # Find LogService
+        response = self.get_request(self.root_uri + self.manager_uri)
+        if response['ret'] is False:
+            return response
+        data = response['data']
+        if 'LogServices' not in data:
+            return {'ret': False, 'msg': "LogServices resource not found"}
+
+        # Find all entries in LogServices
+        logs_uri = data["LogServices"]["@odata.id"]
+        response = self.get_request(self.root_uri + logs_uri)
+        if response['ret'] is False:
+            return response
+        data = response['data']
+        for log_svcs_entry in data[u'Members']:
+            response = self.get_request(self.root_uri + log_svcs_entry[u'@odata.id'])
+            if response['ret'] is False:
+                return response
+            _data = response['data']
+            log_svcs_uri_list.append(_data['Entries'][u'@odata.id'])
+
+        # For each entry in LogServices, get log name and all log entries
+        for log_svcs_uri in log_svcs_uri_list:
+            logs = {}
+            list_of_log_entries = []
+            response = self.get_request(self.root_uri + log_svcs_uri)
+            if response['ret'] is False:
+                return response
+            data = response['data']
+            logs['Description'] = data['Description']
+            # Get all log entries for each type of log found
+            for logEntry in data[u'Members']:
+                entry = {}
+                # I only extract some fields - Are these entry names standard?
+                entry['Name'] = logEntry[u'Name']
+                entry['Created'] = logEntry[u'Created']
+                entry['Message'] = logEntry[u'Message']
+                entry['Severity'] = logEntry[u'Severity']
+                list_of_log_entries.append(entry)
+            logs['entries'] = list_of_log_entries
+            list_of_logs.append(logs)
+
+        # list_of_logs[logs{list_of_log_entries[entry{}]}]
+        return {'ret': True, 'list_of_logs': list_of_logs}
+
+    def clear_logs(self):
+        result = {}
```

Comment:
Should this be `response` since that is what is returned? I don't see `result` used within this function.